### ISSUE TITLE
[BUG] Matchers interface with reverse order parameters

### ISSUE THREAD

**Issue by marco-bressi-deltatre** [microsoft/playwright#8588]
**Context:**
- Playwright Version: 1.14.1
- Operating System: All
- Node.js version: 14.17.1
- Browser: All

**Matchers interface** (toHaveAttribute and toHaveCSS)
```typescript

/**
* Asserts element's attributes `name` matches expected value.
*/
toHaveAttribute(expected: string | RegExp, name: string, options?: { timeout?: number }): Promise<R>;

/**
* Asserts element's computed CSS property `name` matches expected value.
*/
toHaveCSS(expected: string | RegExp, name: string, options?: { timeout?: number }): Promise<R>;
```

**Relative documentation**
```
expect(locator).toHaveAttribute(name, value)#
name: <string> Attribute name
value: <string|RegExp> Attribute value

expect(locator).toHaveCSS(name, value)
name: <string> CSS property name
value: <string|RegExp> CSS property value
````

**Describe the bug**
The interface has the parameters in a reverse order. As the documentation said, the `value` parameter should accept string and RegExp values and the `name` param a simple string.

**Comment by pavelfeldman** [comment]
Where is this code coming from? It does not look like @playwright/test

**Comment by marco-bressi-deltatre** [comment]
> Where is this code coming from? It does not look like @playwright/test

From testExpected type definition:
node_modules/@playwright/test/types/testExpect.d.ts

**Comment by marco-bressi-deltatre** [comment]
Hi @pavelfeldman, any news on this?

**Comment by JoelEinbinder** [comment]
This got fixed by #8314
